app/models/recommendation.rb

```diff
@@ -60,6 +60,10 @@ def unchallenged?
     !challenged
   end
 
+  def accepted?
+    review_complete? && challenged == false
+  end
```

Comment:
Should probably add a test for this (I was about to write one, honest 😆)

Reply:
Rebase with main and you get free tests from ben